apps/src/blockly/utils.js

```diff
@@ -1,16 +1,24 @@
 import _ from 'lodash';
 import {SOUND_PREFIX} from '@cdo/apps/assetManagement/assetPrefix';
 
+// Considers an attribute true only if it is explicitly set to 'true' (i.e. defaults to false if unset).
+export const FALSEY_DEFAULT = attributeValue => attributeValue === 'true';
+
 /**
- * Reads a boolean attribute from an XML element.
- * @param {Element} xmlElement - The XML element from which the attribute will be read.
- * @param {string} attribute - The name of the attribute to be read from the XML element.
- * @returns {boolean} True if the attribute value is exactly 'true', otherwise false.
- * If we ever need to return true for unset attributes, we can update this function.
+ * Reads a boolean attribute from an XML element and determines its value based on a callback function.
+ * The callback function determines how we interpret the attribute value as a boolean.
+ * @param {Element} xmlElement - The XML element from which to read the attribute.
+ * @param {string} attribute - The name of the attribute to read from the XML element.
+ * @param {function(string): boolean} [callback=FALSEY_DEFAULT] - A callback function that takes the attribute value as a string and returns a boolean.
+ * @returns {boolean} The boolean value of the attribute as determined by the callback function.
  */
-export function readBooleanAttribute(xmlElement, attribute) {
+export function readBooleanAttribute(
```

Comment:
Before I realized we don't need to worry about reading `deletable` from the block XML and adding it to the mutator, I thought we needed a `TRUTHY_DEFAULT` option here. This commit shows where I removed the code: https://github.com/code-dot-org/code-dot-org/pull/55346/commits/c0b70dd8a33425dbea0c40bf13c43f62ab0559a5

I could also revert this refactor, but figured I'd ask first.